### PR TITLE
Listens for suspend events from jitsi-power-monitor on postis channel.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: conference.js [hunk 1]
```diff
@@ -1950,6 +1954,23 @@ export default {
...
+        // listens for messages about suspend from power-monitor
+        const transport = new Transport({
+            backend: new PostMessageTransportBackend({
+                postisOptions: { scope: 'jitsi-power-monitor' }
+            })
+        });
+
+        transport.on('event', event => {
+            if (event && event.name === 'power-monitor' && event.event === 'suspend') {
+                room.eventEmitter.emit(JitsiConferenceEvents.SUSPEND_DETECTED);
```
Comment (hristoterezov) [inline conference.js]: Isn't this the eventEmitter of `JitsiConference` instance from lib-jitsi-meet?

If yes, isn't this a little bit dirty?

File: conference.js [hunk 2]
```diff
@@ -1950,6 +1954,23 @@ export default {
             APP.API.notifySuspendDetected();
         });
 
+        // listens for messages about suspend from power-monitor
+        const transport = new Transport({
```
Comment (hristoterezov) [inline conference.js]: Have you considered adding this code somewhere else? Maybe in the react folder?

Comment (damencho) [reply conference.js]: Yep, but not sure where this can fit, ideas?

File: conference.js (hunk 1 above)
Comment (damencho) [reply conference.js]: Nope, we had added a new path to detect suspended PC and we make sure the same event is fired, so all listening for suspending will be notified.

**Pull request merged**